### DISCUSSION TITLE
QucsStudio 3.x Support!!

### DISCUSSION THREAD

**Discussion by Paulo-D2000** [thomaslepoix/Qucs-RFlayout#30] [Ideas]
Hey I made this little converter from QucsStudio 3.x to Qucs 0.0.x Still some bugs left... :) 

https://github.com/Paulo-D2000/QucsStudioConverter

**Comment by thomaslepoix** [comment]
Hey! Nice, some of my users will be grateful! :)

**Comment by thomaslepoix** [comment]
My bad, I anounnced your tool on Discord but forgot to do so in the README! done :)
